.travis.yml

```diff
@@ -7,7 +7,9 @@ before_script:
  - chmod +x deploy.sh
 install:
   - pip3 install -r requirements.txt
+  - pip3 install pylint
 script:
   - python -m unittest discover tests -v
+  - pylint ./app -v
```

Comment:
Can you explain the `-v`

Reply:
I thought it makes the output verbose, similar to the testing command. But found that -v has no effect (but code runs), so I'll remove it. But if I use any other code like -h -t etc then script doesn't run.